common/src/main/java/com/wynntils/wynn/item/parsers/WynnItemMatchers.java

```diff
@@ -107,6 +107,8 @@ public static boolean isUnidentified(ItemStack itemStack) {
     public static boolean isEmeraldPouch(ItemStack itemStack) {
         return itemStack instanceof EmeraldPouchItemStack
```

Comment:
Imo lets refactor this!

1, If itemStack is EmeraldPouchStack, return true.

2, Otherwise return Item is (Golden Shovel or Diamond Axe) AND name is "...". Also, maybe add a comment that there are two possible items because of the pickup "plus texture".

Reply:
Done! Lmk if you wanted the EmeraldPouchStack in a conditional, still using an or operator

Reply:
> Done! Lmk if you wanted the EmeraldPouchStack in a conditional, still using an or operator 

I think we should return early if the instanceof matches. So I think do the "conditional" way.